packages/app-lib/src/api/minecraft_skins/png_util.rs

```diff
@@ -283,6 +314,28 @@ fn copy_rect_mirror_horizontally<PixelType: NoUninit + AnyBitPattern>(
     }
 }
 
+/// Strips alpha from a rectangle of pixels within `texture_buf`
+/// whose top-left corner is at `(x1, y1)` and bottom-right corner is at `(x2 - 1, y2 - 1)`.
+fn strip_alpha(
+    texture_buf: &mut [u8],
+    texture_info: &png::Info,
+    texture_color_type: png::ColorType,
+    x1: usize,
+    y1: usize,
+    x2: usize,
+    y2: usize,
+) {
+    if texture_color_type.samples() != 4 {
+        return;
+    }
```

Comment:
At this point, a valid skin texture buffer can be in one of the following formats: Gray (1 channel), Gray + Alpha (2 channels), RGB (3 channels), or RGBA (4 channels). Therefore, this function should handle the Gray + Alpha case in addition to RGBA.